### ISSUE TITLE
Provide binaries for `dioxus-cli`

### ISSUE THREAD

**Issue by Gobidev** [DioxusLabs/dioxus#1490]
## Specific Demand

The `dioxus-cli` takes a long time to build on slower machines, so it would be nice to have official binaries available for download.

## Implement Suggestion

The build and release could be automated with GitHub Actions.

**Comment by jkelleyrtp** [comment]
This would be a great contribution but is also top of our list!

**Comment by esimkowitz** [comment]
This is already available, though is currently blocked by a bug. See #1431 for the fix

**Comment by jkelleyrtp** [comment]
Took a while but should now be fixed.
